Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM python:3.10
+ENV PYTHONDONTWRITEBYTECODE=1
 ENV PYTHONUNBUFFERED=1
 WORKDIR /app
 COPY . /app
 
 RUN apt update -y && apt install -y libldap2-dev libsasl2-dev gettext
-
-RUN pip install poetry && poetry install && poetry run python manage.py compilemessages
```

Comment:
If we don't run `compilemessages` here, we'd need to do it in the `docker-compose-yml` files from our deployed environments, right? 
I think we'd rather have the docker image prepared in the `Dockerfile` as much as possible so that our test and prod environment have less risk of growing differences.
The `docker-compose-yml` file in this repo is only for a local dev environment.